restore-sql-server-transaction-logs/Function/main.py

```diff
@@ -0,0 +1,258 @@
+import functions_framework
+import google.auth
+import google.auth.transport.requests
+import googleapiclient.discovery
+import logging
+import os
+from time import sleep
+from google.cloud import storage
+from google.cloud import error_reporting
+
+
+def get_environment_variables(logger) -> [str, str, int]:    
+
+    use_fixed_file_name_format = os.environ.get('USE_FIXED_FILE_NAME_FORMAT')
+    processed_bucket_name = os.environ.get('PROCESSED_BUCKET_NAME')        
+    max_operation_fetch_time_seconds = os.environ.get('MAX_OPERATION_FETCH_TIME_SECONDS')
+
+    if (not use_fixed_file_name_format or use_fixed_file_name_format.lower() not in ["true", "false"]):
+        logger.info("Environment variable USE_FIXED_FILE_NAME_FORMAT must be either True or False. Found no valid value. Assumed False.")            
+        use_fixed_file_name_format = "False"
+
+    doDefault_max_operation_fetch_time_seconds = False
+
+    if max_operation_fetch_time_seconds:
+        try:    
+            max_operation_fetch_time_seconds = int(max_operation_fetch_time_seconds)
+            if (max_operation_fetch_time_seconds<3 or max_operation_fetch_time_seconds>120):
+                doDefault_max_operation_fetch_time_seconds = True
+        except ValueError:
+            doDefault_max_operation_fetch_time_seconds = True            

```

Comment:
Worth logging this condition?

Reply:
I agree and we do log this in lines 34-36:

if doDefault_max_operation_fetch_time_seconds:

        max_operation_fetch_time_seconds = 30

        logger.info("Environment variable MAX_OPERATION_FETCH_TIME_SECONDS must be an integer greater that 2 and less thant 120 seconds. Found no valid value. Defaulted to 30 seconds.")
        

Or do you have something else in mind?

Reply:
Ah I missed that. LGTM